init_mysql.sql

```diff
@@ -0,0 +1,354 @@
+use araknemu;
```

Comment:
Do not force database name here. Removing this intruction permit to create tables into any database using :
```
mysql -u my_user my_database_name < init_mysql.sql
```